com.unity.multiplayer.mlapi/Runtime/Core/RPCQueue/QueueHistoryFrame.cs

```diff
@@ -1,7 +1,7 @@
 using System.Collections.Generic;
 using MLAPI.Serialization.Pooled;
 
-namespace MLAPI
+namespace MLAPI.Messaging
```

Comment:
if you're making this change, you may consider moving this source file to /Messaging folder too — or reverting this namespace change?